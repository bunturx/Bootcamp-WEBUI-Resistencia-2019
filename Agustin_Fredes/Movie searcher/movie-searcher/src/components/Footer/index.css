.footer {
  background: rgb(29, 29, 29);
  height: 4%;
  display: flex;
  justify-content: space-between;
  align-items: center;
  color: white;
}
.footer > * {
  margin: 0% 1% 0% 1%;
  font-size: 2.3vmin;
}
.footerSpanG {
  color: rgb(0, 200, 0);
}
.footerSpanR {
  color: rgb(200, 0, 0);
}
