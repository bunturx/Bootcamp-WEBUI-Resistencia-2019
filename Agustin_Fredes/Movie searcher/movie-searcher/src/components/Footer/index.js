import React from "react";
import "./index.css";

const Footer = props => {
  return (
    <footer className="footer">
      <p>
        Bootcamp <span className="footerSpanG">Globant</span> Proyecto Final
      </p>
      <p>
        Creado por <span className="footerSpanR">Agustin Fredes</span>
      </p>
    </footer>
  );
};

export default Footer;
