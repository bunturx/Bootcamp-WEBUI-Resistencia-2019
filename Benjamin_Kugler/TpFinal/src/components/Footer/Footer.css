footer{
    height: 50px;
    width: 100%;
    background-color: lightgray;
    margin: 0;
    padding: 0;
    display: flex;
    align-items: center;
}