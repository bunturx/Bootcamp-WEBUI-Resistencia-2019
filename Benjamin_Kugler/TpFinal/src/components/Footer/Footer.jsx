import React from 'react';

const Footer = () => (
    <footer>
        <h3> Globant Bootcamp July 2019</h3>
    </footer>
);
export default Footer;