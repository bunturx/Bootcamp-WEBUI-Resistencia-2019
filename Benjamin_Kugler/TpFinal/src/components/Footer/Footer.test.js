import React from 'react';
import Footer from './Footer.jsx';
import { configure, shallow, render, mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter(), disableLifecycleMethods: true })


function setup() {
    let wrapper = shallow(<Footer />);
    return wrapper;
}

describe("<Footer />", () => {
    it("Should match Snapshot", () => {
        const component = setup();
        expect(component).toMatchSnapshot();
    })
})