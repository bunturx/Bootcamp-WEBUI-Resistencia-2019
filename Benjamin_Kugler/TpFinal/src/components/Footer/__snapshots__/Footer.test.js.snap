// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should match Snapshot 1`] = `ShallowWrapper {}`;
